### PR TITLE
[HOPSWORKS-1525] NodeManager auto discover hardware capabilities

### PR DESCRIPTION
**Pull request by kouzant**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1525

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -85,9 +85,13 @@
 default['hops']['yarn']['vpmem_ratio']         = "50.1"
 default['hops']['yarn']['vmem_check']          = false
 end
+default['hops']['yarn']['pmem_check']                  = true
 
+default['hops']['yarn']['detect-hardware-capabilities'] = "false"
+default['hops']['yarn']['logical-processors-as-cores'] = true
+default['hops']['yarn']['pcores-vcores-multiplier']    = "0.9"
+default['hops']['yarn']['system-reserved-memory-mb']   = "-1"
```
Comment (SirOibaf) [inline attributes/default.rb]: Could you please tabulate?

File: attributes/default.rb [hunk 2]
```diff
@@ -85,9 +85,13 @@
 default['hops']['yarn']['vpmem_ratio']         = "50.1"
 default['hops']['yarn']['vmem_check']          = false
 end
+default['hops']['yarn']['pmem_check']                  = true
 
+default['hops']['yarn']['detect-hardware-capabilities'] = "false"
+default['hops']['yarn']['logical-processors-as-cores'] = true
```
Comment (SirOibaf) [inline attributes/default.rb]: this should be a string.

File: attributes/default.rb [hunk 3]
```diff
@@ -85,9 +85,13 @@
 default['hops']['yarn']['vpmem_ratio']         = "50.1"
 default['hops']['yarn']['vmem_check']          = false
 end
+default['hops']['yarn']['pmem_check']                  = true
```
Comment (SirOibaf) [inline attributes/default.rb]: this should be a string

Comment (kouzant) [reply attributes/default.rb]: I also added it to metadata as it was missing

**Pull request merged**